packages/perseus-editor/src/hint-editor.tsx

```diff
@@ -89,7 +89,10 @@ export class HintEditor extends React.Component<HintEditorProps> {
     };
 
     serialize: (options?: any) => any = (options: any) => {
-        return this.editor.current?.serialize(options);
+        return {
+            ...this.editor.current!.serialize(options),
+            replace: this.props.replace ?? undefined,
```

Comment:
Why is `?? undefined` needed here?

Reply:
Because `this.props.replace` can be `null`, but a `Hint`'s `replace` is defined as `replace?: boolean;`. One of my gripes with so many places we have `X | null | undefined`. 

Is there a better way?

Reply:
No, this seems fine! Thanks for explaining.